llvm/test/CodeGen/X86/avx512fp16-fminimum-fmaximum.ll

```diff
@@ -28,35 +28,17 @@ define half @test_fminimum(half %x, half %y) {
 define <8 x half> @test_fminimum_scalarize(<8 x half> %x, <8 x half> %y) "no-nans-fp-math"="true" "no-signed-zeros-fp-math"="true" {
```

Comment:
The `_scalarize` should be removed now.

@e-kud What's these tests testing for? If in a NNan, Nsz situation, we can generate VMIN/MAXPH directly. Or you actually wanted `"no-nans-fp-math"="false" "no-signed-zeros-fp-math"="false"`?

Reply:
As you've said we test that only VMIN/VMAX is enough. Otherwise we  need to deal with `+/-0` using `fpclass` and `NaN`s with `vcmpunord`. It's called scalarize because already scalarized version of FMINIMUM comes to lowering. Probably this is the reason why it has no much support. Let me see what's going wrong and why we stop vectorizing it.

Reply:
What do you mean should be removed? Based on the test name, the attributes should not be here. Also, should migrate to using the instruction flags. In any case this patch shouldn't do anything with this function, other than update it

Reply:
The codegen was scalarized for some reason, and it's not ture with this patch. So keeping this name is confusing. It's not important since @e-kud kindly help investigating it.

Reply:
So the situation is the following: `FMINIMUM` and `FMAXIMUM` are set to `Expand` through `setF16Action` lambda. And I didn't set `v8f16` separately to `Custom` because I saw `SELECT` inside `setF16Action`. And since `FMINIMUM` lowering requires support of `SELECT` I decided that we need to support `SELECT` for `v8f16` first. Now I see, that despite of `SELECT` inside `setF16Action`, we overwrite it later with `Custom` for `v8f16`.

When I've added `Custom` for `FMINIMUM` I've seen the desired result
```asm
test_fminimum_scalarize:                # @test_fminimum_scalarize 
        .cfi_startproc                                             
# %bb.0:                                                           
        vminph  %xmm1, %xmm0, %xmm0                                
        retq                                                       
```
Let me support it with fixing test naming to illustrate the attributes